src/transactions/feed/TransactionFeedItemImage.tsx

```diff
@@ -20,12 +21,17 @@ type Props =
   | {
       status: TransactionStatus
       transactionType: 'TokenTransferV3'
+      transactionSubtype: TokenTransactionTypeV2
       recipient: Recipient
     }
   | {
       status: TransactionStatus
       transactionType: 'TokenApproval'
     }
+  | {
+      status: TransactionStatus
+      transactionType: 'JumpstartDeposit'
+    }
 
 function TransactionFeedItemImage(props: Props) {
```

Comment:
maybe we can can use `transfer.type` instead of `__typename` as a `transactionType` because looks like it's more detailed? then we can not bother with introducing a subtype.

Reply:
thanks for the detailed analysis! 
yes, I agree that this is a less destructive approach to reach the goal.